flask_security/ldap/datastore.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""
+    flask_security.ldap.datastore
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Flask-Security ldap datastore module
+
+    :copyright: (c) 2012 by Matt Wright.
+    :license: MIT, see LICENSE for more details.
+"""
+
+import six
+import ldap
+
+from ..datastore import SQLAlchemyUserDatastore
+from ..utils import config_value
+
+
+class LDAPUserDatastore(SQLAlchemyUserDatastore):
+    def __init__(self, db, user_model, role_model):
```

Comment:
what does the model mean in this case? The docs say that the model must have a password: https://pythonhosted.org/Flask-Security/models.html but that should not be the case when using LDAP.